Data/IntMap.hs

```diff
@@ -31,20 +32,6 @@
 -- >  import Data.IntMap (IntMap)
 -- >  import qualified Data.IntMap as IntMap
 --
--- The implementation is based on /big-endian patricia trees/.  This data
--- structure performs especially well on binary operations like 'union'
--- and 'intersection'.  However, my benchmarks show that it is also
--- (much) faster on insertions and deletions when compared to a generic
--- size-balanced map implementation (see "Data.Map").
---
---    * Chris Okasaki and Andy Gill,  \"/Fast Mergeable Integer Maps/\",
---      Workshop on ML, September 1998, pages 77-86,
---      <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.37.5452>
---
---    * D.R. Morrison, \"/PATRICIA -- Practical Algorithm To Retrieve
```

Comment:
Isn't this data structure still based on *ideas* from the PATRICIA paper? If so, the citation should remain. It may be that nothing remains here of Okasaki and Gill, in which case it would be reasonable to delete that reference; I'll leave that up to you.